man/systemd-repart.xml

```diff
@@ -265,6 +265,15 @@
         and graphic illustrating the changes applied.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--json=</option><replaceable>MODE</replaceable></term>
+
+	<listitem><para>Shows output formatted as JSON. Expects one of <literal>short</literal> (for the
+	shortest possible output without any redundant whitespace or line breaks), <literal>pretty</literal>
+	(for a pretty version of the same, with indentation and line breaks) or <literal>off</literal> (to turn
+	off json output).</para></listitem>
+      </varlistentry>
```

Comment:
please follow usual indentation

Reply:
Oh, sorry for this, I did not notice the tab my editor added!